### ISSUE TITLE
Number rounding error in certain invoices

### ISSUE THREAD

**Issue by vwirta** [Clinical-Genomics/cg#1220]
## Description

(I am not sure this issue is in the correct repo)

In certain invoices the sum of the KTH and KI do not match the sum included on the KI invoice.
This is due to rounding error somewhere.
See for example sample ACC7985A1 on invoice 627 https://cg-statusdb.scilifelab.se/invoices/627

## Suggested solution

Include one decimal or recalculate the total from the KTH + KI costs without decimals.

## This can be closed when

Sum of KTH and KI invoices match the total.

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by Mropat** [comment]
This is a serious bug, needs priority assigned

**Comment by vwirta** [comment]
Not a serious bug, but I now need to correct for this manually on each invoice where I observe this.
From a financial point of view this is peanuts, but if not corrected the invoices will be blocked at the invoicing units and I need to manually prepare new corrected specifications.

**Comment by patrikgrenfeldt** [comment]
@vwirta Could you provide some screenshots to make it super clear?

**Comment by vwirta** [comment]
I created a temporary invoice specification 827 to show what happens
https://cg-statusdb.scilifelab.se/invoices/827
I've also attached a screenshot with the relevant sections of the invoice specification.

In this case the analysis is EXOKTTR040 with a total cost of 7123 kr. This is fetched from the Application Version table in statusDB. The Application table defines the percentage of cost that belongs to KTH, ie 63% in this case.
 
The KI and KTH costs are according to the invoice 2636 kr  and 4488 kr, respectively. But these two add up to 7124 kr, ie these is a difference of 1 kr. 

I am fine with either one of the proposed solutions. 
Solution 1: Use 1 decimal: 2635.5 kr + 4487.5 kr = 7123 kr
Solution 2: Derive the total column by summing KI and KTH costs 2636 + 4488 = 7124 kr

fyi @patrikgrenfeldt 

<img width="842" alt="Screenshot 2022-04-19 at 22 36 42" src="https://user-images.githubusercontent.com/2646553/164091363-7ae2c47e-fbeb-44b3-9ccc-b5e1f06af616.png">
